metadata-ingestion/src/datahub/ingestion/source/looker/looker_lib_wrapper.py

```diff
@@ -75,10 +87,16 @@ def __init__(self, config: LookerAPIConfig) -> None:
                 "Failed to initialize Looker client. Please check your configuration."
             ) from e
 
-    def get_client(self) -> Looker31SDK:
-        return self.client
+        self.client_stats = LookerAPIStats()
+
+    @staticmethod
+    def __fields_mapper(fields: Union[str, List[str]]) -> str:
```

Comment:
nit - double leading underscore has a special meaning in python; a single underscore would be stylistically correct

Reply:
ack